.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: true
 
 language: node_js
 node_js:
-  - "10.9"
+  - "10.13"
```

Comment:
Might you be using an old version? Please pull the master branch into this branch first

Reply:
Yes that was my bad.